Parsers/ASimAuthentication/Parsers/vimAuthenticationGithub.yaml

```diff
@@ -0,0 +1,133 @@
+ParserName: vimAuthenticationGithub
+EquivalentBuiltInParser: _Im_Authentication_Github
+ParserParams:
+  - Name: starttime
+    Type: datetime
+    Default: datetime(null)
+  - Name: endtime
+    Type: datetime
+    Default: datetime(null)
+  - Name: actor_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: targetappname_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: srcipaddr_has_any_prefix
+    Type: dynamic
+    Default: dynamic([])
+  - Name: srchostname_has_any
+    Type: dynamic
+    Default: dynamic([])
+  - Name: eventtype_in
+    Type: dynamic
+    Default: dynamic([])
+  - Name: eventresultdetails_in
+    Type: dynamic
+    Default: dynamic([])
+  - Name: eventresult
+    Type: string
+    Default: '*'
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+     let parser = (
+     starttime:datetime=datetime(null), 
+     endtime:datetime=datetime(null), 
+     username_has_any:dynamic = dynamic([]),
+     targetappname_has_any:dynamic = dynamic([]),
+     srcipaddr_has_any_prefix:dynamic = dynamic([]),
+     srchostname_has_any:dynamic = dynamic([]),
+     eventtype_in:dynamic =dynamic([]),
+     eventresultdetails_in:dynamic = dynamic([]),
+     eventresult:string = '*',
+     disabled:bool=false
+     )
+     {
+     let EventResultDetailslookup=datatable(EventOriginalResultDetails:string,EventResult:string,EventType:string,EventSeverity:string )
+     ["user.login","Success","logon","Informational",
+     "user.logout","Success","logoff","Informational",
+     "user.two_factor_challenge_success","Success","logon","Informational",
+     "user.two_factor_challenge_failure","Failure","logon","Informational",
+     "user.two_factor_requested","Success","logon","Informational",
+     "user.failed_login","Failure","logon","Informational",
+     "user.sign_in_from_unrecognized_device_and_location","Success","logon","High",
+     "user.sign_in_from_unrecognized_device","NA","logon","High",
+     "user.reset_password","Success","logon","Informational",
+     "user.device_verification_requested","Success","logon","Informational",
+     "user.device_verification_requested","Success","logon","Informational",
+     "user.two_factor_recovery_codes_downloaded","Success","logon","Informational",
+     "user.forgot_password","Success","logon","Informational",
+     "user.new_device_used","Success","logon","Informational",
+     "user.promote","Success","Elevate","Informational",
+     ];
+     let GithubAuthenticationEvent=EventResultDetailslookup 
+     |project EventOriginalResultDetails
+     | summarize make_set(EventOriginalResultDetails);r.promote","Success","Elevate","Informational",
+     ];
+     GitHubAuditLogPolling_CL
+     |where not(disabled)
+     | extend TimeGenerated = todatetime(tostring(split(TimeGenerated, '.', 0)[0]))      
+     |where operation_type_s =='authentication'
```

Comment:
GithubAuthenticationEvent condition check missing in vim file. exist in ASIM file